rising/transforms/spatial.py

```diff
@@ -62,9 +62,10 @@ def __init__(self, dims: tuple, keys: tuple = ('data',),
         --------
         :func:`torch.Tensor.rot90`
         """
-        super().__init__(grad=grad, **kwargs)
-        self.dims = dims
-        self.keys = keys
+        super().__init__(grad=grad, num_rots=DiscreteParameter((0, 1, 2, 3)),
```

Comment:
Should be add `num_rots` as a  `DiscreteParameter ` to our init with this default value?